components/combobox/apiExamples/basic.html

```diff
@@ -1,4 +1,6 @@
 <auro-combobox>
+  <span slot="ariaLabel.close">Close combobox</span>
```

Comment:
```suggestion
  <span slot="ariaLabel.close">Close combobox</span>
```

This part name needs to infer it is in the bib. Consumers won't necessarily understand that just be looking at the name. `ariaLabel.bib.close`